README.md

```diff
@@ -31,6 +31,11 @@ Checkout [`ember-sparkles`](https://github.com/LocusEnergy/ember-sparkles) to se
   - [`d3-join`](#d3-join)
   - [`d3-attr`](#d3-attr)
   - [`d3-call`](#d3-call)
+* [Shape helpers](#selection-helpers)
+  - [`d3-arc`](#d3-arc)
```

Comment:
Nit, these anchors won't work because the ones generated from the headings will be different.